portfolio/src/main/webapp/fetcher.js

```diff
@@ -1,9 +1,10 @@
 //quick fetch function 
-async function strFetch(){
-    const gson = new Gson();
-    const servletText = (await fetch('/strservlet')).text(); //get the servlet text
+async function strFetch(){  
+  const servletText = await (await fetch('/strservlet')).json(); //get the servlet text as json
 
-    const servletJSON = gson.toJSON(servletText) 
+  let randomQuote = ""
+  let randomIdx = Math.floor(Math.random() * servletText.length)
+  randomQuote = await servletText[randomIdx]; //choose a quote from a random index
```

Comment:
Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace